more_itertools/more.py

```diff
@@ -672,8 +672,8 @@ def unique_to_each(*iterables):
     return [list(filter(uniques.__contains__, it)) for it in pool]
 
 
-def windowed(seq, n, fillvalue=None, step=1):
-    """Return a sliding window of width *n* over the given iterable.
+def windowed(iterable, width, fillvalue=None, step=1):
```

Comment:
I'd prefer if the parameter names didn't change.

Reply:
The discussion of **n** versus **width** seems like a waste of time for me, if you believe that somewhere there is a dude who passes a named parameter **n** to the windowed() - your right to save this argument name.